# camera image topic name , RGB image
camera_image_topic_name: "/camera/color/image_raw"
# ros service topic name to set camera info,
set_camera_info_service_topic_name: "/camera/color/set_camera_info"
# calibration file name under your /home/user_name/calibration_path
calibration_path: "calibration.ini"
#DO NOT CHANGE THE VALUES,
gray_level_precision: 0.7
size_precision: 0.5
pause_at_each_frame: True
# BE SURE THAT PRINTED MARKER CIRCLES HAVE 30MM linear distance between each other
model_points_x: [0.0, 0.03, 0.06, 0.09, 0.12, 0.15, 0.0, 0.03, 0.06, 0.09, 0.12, 0.15, 0.0, 0.03, 0.06, 0.09, 0.12, 0.15, 0.0, 0.03, 0.06, 0.09, 0.12, 0.15, 0.0, 0.03, 0.06, 0.09, 0.12, 0.15, 0.0, 0.03, 0.06, 0.09, 0.12, 0.15]
model_points_y: [0.0, 0.00, 0.00, 0.00, 0.00, 0.00, .03, 0.03, 0.03, 0.03, 0.03, 0.03, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.09, 0.09, 0.09, 0.09, 0.09, 0.09, 0.12,0.12, 0.12, 0.12, 0.12, 0.12, 0.15,0.15, 0.15, 0.15, 0.15, 0.15]
model_points_z: [0.0, 0.00, 0.00, 0.00, 0.00, 0.00, 0.0, 0.00, 0.00, 0.00, 0.00, 0.00, 0.0, 0.00, 0.00, 0.00, 0.00, 0.00, 0.0, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.0, 0.00, 0.00, 0.00, 0.00,0.00]
selected_points_x: [0.03, 0.03, 0.09, 0.12]
selected_points_y: [0.03, 0.12, 0.12, 0.03]
selected_points_z: [0.00, 0.00, 0.00, 0.00]